### PR TITLE
Frontend SignIn/SignUp with FRE User Flow

### PR DESCRIPTION
**Pull request by SealsJ**
Overhauled Front-End with Rachel's Forgot Password, Sign-In, Sign-Up, and Success Pages.

Also Includes FRE Three Stage User Verification Front End Design.

I know since this is a bulk merge with Our tickets, I Did my best to leave detailed comments for the FRE pages.

### CODE REVIEW

**Review by lai-kevin — DISMISSED** [review]

File: fwb/middleware.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { authMiddleware } from "@clerk/nextjs";
 // Please edit this to allow other routes to be public as needed.
 // See https://clerk.com/docs/references/nextjs/auth-middleware for more information about configuring your Middleware
 export default authMiddleware({
-      publicRoutes: ["/"]
+      publicRoutes: ["/","/forgotpassword",'/success',]
```
Comment (dericky1) [inline fwb/middleware.ts:7]: We need to discuss which routes should be public and which should not. I had trouble accessing the sign-in and sign-up page. Clerk would auto redirect me to signing in using their own sign in component.

File: fwb/app/(auth)/(routes)/sign-in/[[...sign-in]]/page.tsx [hunk 2]
```diff
@@ -330,30 +402,100 @@ export default function Page() {
...
+            </div>
+            <div className="boyPhoto">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="140"
+                height="180"
+                viewBox="0 0 150 180"
+                fill="none"
+              >
+                <path
```
Comment (dericky1) [inline fwb/app/(auth)/(routes)/sign-in/[[...sign-in]]/page.tsx:416]: Is there a way to maybe condense this <path> element into something smaller?

File: fwb/app/(auth)/(routes)/sign-up/[[...sign-up]]/page.tsx [hunk 3]
```diff
@@ -1,5 +1,2112 @@
-import { SignUp } from "@clerk/nextjs";
- 
+"use client";
```
Comment (dericky1) [inline fwb/app/(auth)/(routes)/sign-up/[[...sign-up]]/page.tsx:1]: When I go through the sign-up and make a new account after I verify it redirects me back to clerks own sign-in component. The account I created also doesn't get registered into Clerk.

**Review by dericky1 — COMMENTED** [review]
I ran into a lot of issues signing up and signing in. I think we should set up a meeting to discuss them. @lai-kevin @SealsJ

**Comment by lai-kevin** [comment]
For files with only style changes, separate them into a separate pull request so the meaningful stuff doesn't get buried. It also helps us save time looking at the changed files

**Comment by lai-kevin** [comment]
I had the same issues as Derrick with the signup but its okay to merge since the project is still early in development. It could be useful to have other feature branches work off of the frontend code here particularly with the user api.

**Comment by SealsJ** [comment]
Sign-in / Sign-up Flows need these envrionmental variables as well to avoid Clerk Components / user Database issues:

NEXT_PUBLIC_CLERK_SIGN_IN_URL=/sign-in
NEXT_PUBLIC_CLERK_SIGN_UP_URL=/sign-up
NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL=/explore
NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL=/success

**Pull request closed without merging**